server/master/src/main/java/org/apache/accumulo/master/tableOps/bulkVer2/BulkImportMove.java

```diff
@@ -109,41 +104,26 @@ private void moveFiles(String fmtTid, Path sourceDir, Path bulkDir, Master maste
 
     int workerCount = master.getConfiguration().getCount(Property.MASTER_BULK_RENAME_THREADS);
     SimpleThreadPool workers = new SimpleThreadPool(workerCount, "bulkDir move");
-    List<Future<Exception>> results = new ArrayList<>();
+    List<Future<Boolean>> results = new ArrayList<>();
 
     for (Map.Entry<String,String> renameEntry : renames.entrySet()) {
       results.add(workers.submit(() -> {
-        try {
-          final Path originalPath = new Path(sourceDir, renameEntry.getKey());
-          boolean success;
-          String errorMsg = "";
-          Path newPath = null;
-          try {
-            newPath = new Path(bulkDir, renameEntry.getValue());
-            success = fs.rename(originalPath, newPath);
-            log.trace("tid {} moved {} to {}", fmtTid, originalPath, newPath);
-          } catch (IOException ioe) {
-            errorMsg = ioe.getMessage();
-            success = false;
-          }
-          if (!success) {
-            // TODO must retry or fail fate op if a move failed
-            if (errorMsg.isEmpty())
-              errorMsg = "HDFS rename returned false";
-            log.warn("Could not move: {} to {} {}", originalPath.toString(), newPath, errorMsg);
-          }
-          return null;
-        } catch (Exception e) {
-          return e;
-        }
+        final Path originalPath = new Path(sourceDir, renameEntry.getKey());
+        Path newPath = new Path(bulkDir, renameEntry.getValue());
+        Boolean success = fs.rename(originalPath, newPath);
+        if (success && log.isTraceEnabled())
+          log.trace("tid {} moved {} to {}", fmtTid, originalPath, newPath);
+        return success;
       }));
     }
     workers.shutdown();
     while (!workers.awaitTermination(1000L, TimeUnit.MILLISECONDS)) {}
 
-    for (Future<Exception> ex : results) {
-      if (ex.get() != null) {
-        throw ex.get();
+    for (Future<Boolean> future : results) {
+      if (!future.get()) {
```

Comment:
may want to catch execution exception here.

Reply:
Added a catch.  I assume the actual Exception is in the cause... couldn't confirm it though.